src/js/data.js

```diff
@@ -12,7 +12,8 @@ var data = {
     first_extra:      'sbjs_first_add',
     session:          'sbjs_session',
     udata:            'sbjs_udata',
-    promocode:        'sbjs_promo'
+    promocode:        'sbjs_promo',
+    single:           'sbjs_current',
```

Comment:
Can we have an explanation of the meaning of this `single` here? In a documentation, or maybe in this PR description? 🤔